extensions/job_management/standard/sections/clause_2_conformance.adoc

```diff
@@ -0,0 +1,31 @@
+
+== Conformance
+
+The OGC API - Processes - Part 4 Standard defines the following requirements classes.
+
+The main requirements class is:
+
+* <<rc_job-management,Job Management>>.
+
+This class specifies requirements that any Web API implementing Processes Part 4 must conform with.
+
+The `Job Management` class does not mandate a specific encoding for the job definition.  
+However, the Part 4 extension defines the following conformance class:
+
+* <<rc_ogcapi-processes,OGC API - Processes - Workflow Execute Request>>
+* <<rc_openeo,OpenEO Process Graph>>
+
+The `OGC API - Processes - Workflow Execute Request` class defines that jobs can be created from an OGC API - Processes - Workflow Execute Request.
+
+The `OpenEO Process Graph` class defines that jobs can be created from an OpenEO Process Graph.
```

Comment:
I think later we need to go through the document and replace all occurances of Process GRaph with UDP...